test/fixtures/es-class-assign-property/expected-remove-context-wrap-es6.js

```diff
@@ -0,0 +1,23 @@
+class Foo1 extends React.Component {
+  render() {}
+}
+
+process.env.NODE_ENV !== "production" ? Foo1.contextTypes = {
+  bar1: PropTypes.string
+} : void 0;
+
+process.env.NODE_ENV !== "production" ? Foo1.childContextTypes = {
+  bar1: PropTypes.string
+} : void 0;
+
+class Foo2 extends React.PureComponent {
+  render() {}
+}
+
+process.env.NODE_ENV !== "production" ? Foo1.contextTypes = {
```

Comment:
The point is that React needs the context type key names to provide them to the component. By removing them, you lose the context values, hence, you break the component. At least, that's my understanding of the reality. Have a look at the issue, I think that it's explain in the thread.